project/Dependencies.scala

```diff
@@ -5,7 +5,7 @@ import sbt.Keys._
 object Dependencies {
   object Kalix {
     val ProtocolVersionMajor = 1
-    val ProtocolVersionMinor = 0
+    val ProtocolVersionMinor = 1
```

Comment:
yep, that is what I mean the other day when I said that bumping the minor version required extra work.